### PR TITLE
UI Improvement: Document Template Management

### LINKED ISSUE #3106: UI: Document template page missing BS
BS tags are missing on the document template page.
image

Can I work on this?

### PR DESCRIPTION
**Pull request by thesmallstar**
**Before:**
image

**After:**
image

Fixes #3106

### CODE REVIEW

File: interface/super/manage_document_templates.php [hunk 1]
```diff
@@ -129,97 +129,93 @@
...
+   <body class="body_top">
+      <form method='post' action='manage_document_templates.php' enctype='multipart/form-data'
+         onsubmit='return top.restoreSession()'>
+         <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+         <center>
+            <h2><?php echo xlt('Document Template Management'); ?></h2>
+            <div class="col-6">
+               <div class="mx-auto mt-3">
+                  <div class="card">
+                     <h5 class="card-header">Upload a template</h5>
```
Comment (bradymiller) [inline interface/super/manage_document_templates.php]: needs to maintain translation/escaping:
```
<h5 class="card-header"><?php echo xlt('Upload a Template'); ?></h5>
```

Comment (thesmallstar) [reply interface/super/manage_document_templates.php]: Oh! I missed it. 
On it

File: interface/super/manage_document_templates.php [hunk 2]
```diff
@@ -129,97 +129,93 @@
...
+                     <div class="card-body">
+                        <div class="custom-file">
+                           <input type="hidden" name="MAX_FILE_SIZE" value="250000000" />
+                           <input type="file"  name="form_file" size="40" class="custom-file-input" id="customFile">
+                           <label class="custom-file-label" for="customFile">Choose file</label>
```
Comment (bradymiller) [inline interface/super/manage_document_templates.php]: translation/escaping:
```
<label class="custom-file-label" for="customFile"><?php echo xlt('Choose file'); ?></label>
```

File: interface/super/manage_document_templates.php [hunk 3]
```diff
@@ -129,97 +129,93 @@
...
+                           </div>
+                        </div>
+                     </div>
+                  </div>
+               </div>
+            </div>
+            <div class="col-6">
+               <div class="mx-auto mt-3">
+                  <div class="card">
+                     <h5 class="card-header">Download or Delete a Template</h5>
```
Comment (bradymiller) [inline interface/super/manage_document_templates.php]: translation/escaping:
```
<h5 class="card-header"><?php echo xlt('Download or Delete a Template'); ?></h5>
```

**Comment by bradymiller** [comment]
hi @thesmallstar , Just noted a couple minor issue on review. Should be ready for codebase after you address those. thanks! -brady

File: interface/super/manage_document_templates.php [hunk 4]
```diff
@@ -129,97 +129,93 @@
...
+   <head>
+      <title><?php echo xlt('Document Template Management'); ?></title>
+      <?php Header::setupHeader(); ?>
+      <style type="text/css">
+         .dehead { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:bold }
+         .detail { color:var(--black); font-family:sans-serif; font-size:10pt; font-weight:normal }
+      </style>
+   </head>
+   <body class="body_top">
+      <form method='post' action='manage_document_templates.php' enctype='multipart/form-data'
```
Comment (stu01509) [inline interface/super/manage_document_templates.php]: I think we need to add the container class.

Comment (bradymiller) [reply interface/super/manage_document_templates.php]: agreed

**Comment by bradymiller** [comment]
hi @thesmallstar , Code looks good and bringing tin codebase. I'll fix up the psr2 on a subsequent commit. Thanks for the improvements! -brady

**Pull request merged**